packages/@o3r/amaterasu/amaterasu-dodo/project.json

```diff
@@ -14,16 +14,9 @@
       "dependsOn": ["compile"]
     },
     "compile": {
-      "executor": "@nx/js:tsc",
+      "executor": "nx:run-script",
       "options": {
-        "main": "packages/@o3r/amaterasu/amaterasu-dodo/src/index.ts",
-        "tsConfig": "packages/@o3r/amaterasu/amaterasu-dodo/tsconfig.build.json",
-        "outputPath": "packages/@o3r/amaterasu/amaterasu-dodo/dist",
-        "assets": [
-          "packages/@o3r/amaterasu/amaterasu-dodo/{builders,extractors,schematics}/**/templates/**",
```

Comment:
I think some of these assets are not part of build:source cpy

Reply:
I don't see either `builders`, `extractors` or `schematics` folders in `amaterasu-dodo`